### PR TITLE
Fix code-links containing spaces, API3

### PR DESCRIPTION
**Pull request by Josh-Cena**
Last part to fix wrong macro usage that results in non-code-entities being code formatted.

### CODE REVIEW

File: files/en-us/webassembly/reference/numeric/index.md [hunk 1]
```diff
@@ -10,96 +10,96 @@ WebAssembly numeric instructions.
...
+- [Truncate (float to int)](/en-US/docs/WebAssembly/Reference/Numeric/Truncate_float_to_int)
   - : Convert (truncate fractional part) floating points to integers.
-- [`Reinterpret`](/en-US/docs/WebAssembly/Reference/Numeric/Reinterpret)
+- [Reinterpret](/en-US/docs/WebAssembly/Reference/Numeric/Reinterpret)
   - : Reinterpret the bytes of integers as floating points and vice versa.
 
 ## Floating point specific instructions
 
-- [`Min`](/en-US/docs/WebAssembly/Reference/Numeric/Min)
+- [Min](/en-US/docs/WebAssembly/Reference/Numeric/Min)
```
Comment (estelle) [inline files/en-us/webassembly/reference/numeric/index.md:63]:
```suggestion
- [Minimum](/en-US/docs/WebAssembly/Reference/Numeric/Min)
```
or `min`

File: files/en-us/webassembly/reference/numeric/index.md [hunk 2]
```diff
@@ -10,96 +10,96 @@ WebAssembly numeric instructions.
...
   - : Get the lower of two numbers.
-- [`Max`](/en-US/docs/WebAssembly/Reference/Numeric/Max)
+- [Max](/en-US/docs/WebAssembly/Reference/Numeric/Max)
```
Comment (estelle) [inline files/en-us/webassembly/reference/numeric/index.md:65]:
```suggestion
- [Maximum](/en-US/docs/WebAssembly/Reference/Numeric/Max)
```

File: files/en-us/webassembly/reference/numeric/index.md [hunk 3]
```diff
@@ -10,96 +10,96 @@ WebAssembly numeric instructions.
...
   - : Get the higher of two numbers.
-- [`Nearest`](/en-US/docs/WebAssembly/Reference/Numeric/Nearest)
+- [Nearest](/en-US/docs/WebAssembly/Reference/Numeric/Nearest)
```
Comment (estelle) [inline files/en-us/webassembly/reference/numeric/index.md:67]: `nearest` or "Nearest integer"

File: files/en-us/webassembly/reference/numeric/index.md [hunk 4]
```diff
@@ -10,96 +10,96 @@ WebAssembly numeric instructions.
...
   - : Round a number to the nearest integer.
-- [`Ceil`](/en-US/docs/WebAssembly/Reference/Numeric/Ceil)
+- [Ceil](/en-US/docs/WebAssembly/Reference/Numeric/Ceil)
```
Comment (estelle) [inline files/en-us/webassembly/reference/numeric/index.md:69]: `ceil` or "Ceiling"

File: files/en-us/webassembly/reference/numeric/index.md [hunk 5]
```diff
@@ -10,96 +10,96 @@ WebAssembly numeric instructions.
...
   - : Round up a number.
-- [`Floor`](/en-US/docs/WebAssembly/Reference/Numeric/Floor)
+- [Floor](/en-US/docs/WebAssembly/Reference/Numeric/Floor)
```
Comment (estelle) [inline files/en-us/webassembly/reference/numeric/index.md:71]: changing this to `floor` is optional

**Review by estelle — APPROVED** [review]
A few fixes suggested. Approving to not hold you up.

**Comment by estelle** [comment]
oops, merged accidentally.

**Comment by Josh-Cena** [comment]
The "Touch events overview" is a typo and the "animatable CSS properties" I'll trust your call, but I would prefer to keep the Wasm instruction titles as-is because that aligns with the linked pages.

**Pull request merged**